ee/modules/160-multitenancy-manager/templates/user-resources/user-resources-templates.yaml

```diff
@@ -1,14 +1,16 @@
 {{- define "project.namespace" }}
 {{ $name := index . 0 }}
 {{ $metadata := index . 1 }}
+{{ $dedicatedNodeName := index . 2 }}
```

Comment:
I think this variable must be renamed

Reply:
This viable is not used. So it must be deleted.

Reply:
Done